src/tools/ping/PingConnection.h

```diff
@@ -110,6 +121,17 @@ struct PingConnection {
         _In_ HQUIC Connection
         );
 
+    //
+    // Constructor for incoming connection with tracker.
+    // Overload token is because otherwise this matches the first constructor
+    // which is for clients.
+    //
+    PingConnection(
+        _In_ PingTracker* Tracker,
+        _In_ HQUIC Connection,
+        _In_ int OverloadToken
```

Comment:
Instead of doing this, remove the default for `DumpResumption` in the client constructor.